lib/internal/bootstrap/loaders.js

```diff
@@ -184,7 +223,7 @@
       if (id === loaderId) {
         return false;
       }
-      return NativeModule.exists(id);
+      return id === cacheId || NativeModule.exists(id);
```

Comment:
Since this is exposed at the moment can't it be handled as any other internal (in its own file)? That way it avoids conditional checks `NativeModule.require` and `NativeModule.nonInternalExists`.

Reply:
@jdalton Good idea, I'll do that